lib/ordersystem.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'takeaway_kitchen'
+require_relative 'text'
+
```

Comment:
I love how clean your code is. Every method is named well and it's easy to understand what each of them do.